src/coreclr/src/vm/readytoruninfo.cpp

```diff
@@ -602,6 +602,18 @@ ReadyToRunInfo::ReadyToRunInfo(Module * pModule, PEImageLayout * pLayout, READYT
         }
     }
 
+    // For format version 4.1 and later, there is an optional inlining table
+    if (IsImageVersionAtLeast(4, 1))
+    {
+        IMAGE_DATA_DIRECTORY* pInlineTrackingInfoDir = FindSection(READYTORUN_SECTION_INLINING_INFO2);
+        if (pInlineTrackingInfoDir != NULL)
+        {
+            const BYTE* pInlineTrackingMapData = (const BYTE*)GetImage()->GetDirectoryData(pInlineTrackingInfoDir);
+            PersistentInlineTrackingMapR2R2::TryLoad(pModule, pInlineTrackingMapData, pInlineTrackingInfoDir->Size,
+                pamTracker, (PersistentInlineTrackingMapR2R2**)&m_pPersistentInlineTrackingMap);
```

Comment:
Nit: This will leak memory if the image has both `READYTORUN_SECTION_INLINING_INFO2` and `READYTORUN_SECTION_INLINING_INFO`. Look for the new one first and if it does not exist, look for the old one second?